documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -45,9 +45,11 @@ Following the convention for pipeline definitions, use a `Jenkinsfile` which res
 fioriOnCloudPlatformPipeline script:this
 ```
 
+`fioriOnCloudPlatform` is a so called _scenario step_ that wraps the [mtaBuild](https://sap.github.io/jenkins-library/steps/mtaBuild/) and the [neoDeploy](https://sap.github.io/jenkins-library/steps/neoDeploy/) steps.
+
 ### Configuration (`.pipeline/config.yml`)
 
-This is a basic configuration example, which is also located in the sources of the project.
+This is a basic configuration example, which is also located in the sources of the project. The configuration refences to the steps wrapped in `fioriOnCloudPlatformPipeline`.
```

Comment:
There is a typo. The new sentence does not say anything to me (clarify ?). Maybe less is more ...

Reply:
With basic piper knowledge I would always expect to see the configuration example match the steps in the pipeline example. Here, I see fioriOnCloudPlatformPipeline in the Jenkinsfile, but in the config I see `mtaBuild` and `neoDeploy`. I just want to make explicit, what we're doing here. Do you have an idea to make that more transparent?

Reply:
typo fixed.

Reply:
> but in the config I [...]

... aahhh, I see. Yes, we have a higher level pipeline step, bundeling low-level steps together. And what we configure are still the low level steps. So we hide the details in the pipeline itself, but we do not do the same - and actually we can't - in the configuration. I did not understand the sentence without your context. Would be good to know how and if others understand it.

Reply:
@SarahNoack

Reply:
Well, at least with this description (and eventually a link to the definition of fioriOnCloudPlatformPipeline) it would be much more obvious what is going on here.
Also I think fioriOnCloudPlatformPipeline is NOT a step but a complete pipeline (in Jenkins UI I still see the three steps when using fioriOnCloudPlatformPipeline). 
What I am still missing is information how to add eg a test step for eg UI5Veri to fioriOnCloudPlatformPipeline .

Reply:
@DerGuteWolf Thanks for the feedback, and noted, we need to add test content. @IKirilova